node/clients/robots.ts

```diff
@@ -0,0 +1,9 @@
+import { ExternalClient, InstanceOptions, IOContext } from '@vtex/api'
+
+export default class Robots extends ExternalClient {
```

Comment:
I tend not to like default exports since they break the majority of reference search plugins